test/core/event_engine/test_suite/tests/dns_test.cc

```diff
@@ -201,6 +201,8 @@ class EventEngineDNSTest : public EventEngineTest {
 EventEngineDNSTest::DNSServer EventEngineDNSTest::dns_server_;
 bool EventEngineDNSTest::skip_end2end_tests_ = false;
 
+#ifndef GRPC_IOS_EVENT_ENGINE_CLIENT
```

Comment:
Please add `// TODO(hork): implement XFAIL for resolvers that don't support TXT or SRV`.

Context: third-party EventEngine implementations might want to only support A/AAAA records as well, and we can't add ifdefs throughout the test suite for every unknown implementation. I have a spec for a system that will allow declarative support (or lack of) in each implementation's build configuration.

Reply:
Sure.
BTW, this disables the LookupHostname cases as well as the new cases need a custom name server.